SQL/database_changelog.txt

```diff
@@ -1,13 +1,21 @@
 Any time you make a change to the schema files, remember to increment the database schema version. Generally increment the minor number, major should be reserved for significant changes to the schema. Both values go up to 255.
 
-The latest database version is 5.17; The query to update the schema revision table is:
+The latest database version is 5.19; The query to update the schema revision table is:
 
-INSERT INTO `schema_revision` (`major`, `minor`) VALUES (5, 18);
+INSERT INTO `schema_revision` (`major`, `minor`) VALUES (5, 19);
 or
-INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (5, 18);
+INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (5, 19);
 
 In any query remember to add a prefix to the table names if you use one.
 
+-----------------------------------------------------
+Version 5.19, 10 November 2021, by WalterMeldron
+Adds an urgent column to tickets for ahelps marked as urgent.
+
+```
+ALTER TABLE `ticket` ADD COLUMN (`urgent` TINYINT(1) UNSIGNED NOT NULL DEFAULT '0');
```

Comment:
In the schema you've put this column between `action` and `message` columns, for this alter to actually do that you need an `AFTER`

```suggestion
ALTER TABLE `ticket` ADD COLUMN `urgent` TINYINT(1) UNSIGNED NOT NULL DEFAULT '0' AFTER `sender`;
```

Reply:
alrighty